r/rfc3339-validator/rfc3339-validator_ubi_8.4.sh

```diff
@@ -0,0 +1,60 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : rfc3339-validator
+# Version       : v0.1.4
```

Comment:
are these all the version(s) that the script was verified with?

Reply:
yes, verified with latest tag version & without passing tag version. Though it works for all the tags.